fec_eregs/templates/regulations/chrome.html

```diff
@@ -37,3 +32,22 @@
   </a>
 </li>
 {% endblock %}
+
+{% block header-secondary %}
+{% endblock %}
+
+{% block drawer-search %}
+    <div id="search" class="search-drawer toc-container hidden">
+        <div class="drawer-header">
+            <h3 class="toc-type">Search</h3>
+        </div><!-- /.drawer-header -->
+        <div class="drawer-content">
+            <form action="{% url 'chrome_search' reg_part %}" method="GET" role="search">
+                <div class="search-box">
+                    <input type="text" name="q" value="{{ q }}" id="search-input" title="Search term"/><button type="submit"><span class="cf-icon cf-icon-search"></span><strong class="icon-text">Search</strong></button>
+                    <input type="hidden" name="version" id="version-hidden" title="Effective version" value="{{active_version}}" />
+                </div><!--/.search-box-->
+            </form>
+        </div><!-- /.drawer-content -->
+    </div> <!--/#search-->
+{% endblock %}
```

Comment:
I added a drawer-search block to the drawer-history block in chrome.html in regulations-site so that I could modify it here, stripping the versioning UI and adding a hidden field to the search form that contains the active version so that the form submission is unchanged.